test/verify/check-users

```diff
@@ -29,8 +29,8 @@ class TestAccounts(MachineCase):
 
     def tearDown(self):
         # ensure that there are no lingering processes after the test which prevent userdel
-        self.machine.execute("pkill -e -u anton 2>/dev/null; pkill -e -u scruffy 2>/dev/null ; pkill -e -u jussi 2>/dev/null || true")
-        self.machine.execute("while pgrep -u anton; do sleep 1; done; while pgrep -u scruffy; do sleep 1; done; while pgrep -u jussi; do sleep 1; done;")
+        self.machine.execute("pkill -e -u anton 2>/dev/null; pkill -e -u scruffy 2>/dev/null ; pkill -e -u jussi 2>/dev/null; pkill -e -u admin 2>/dev/null || true")
+        self.machine.execute("while pgrep -u anton; do sleep 1; done; while pgrep -u scruffy; do sleep 1; done; while pgrep -u jussi; do sleep 1; done; while pgrep -u admin; do sleep 1; done;")
```

Comment:
Is this still actually necessary after commit 2cd6d5acd609a ? It doesn't harm to have it here in addition, but the generic session cleanup code ought to take care of this as well.

Reply:
(ignoring the fact that there is missing space between `/dev/null` and `||` in https://github.com/cockpit-project/cockpit/commit/2cd6d5acd609abf5a35eedad44b6cac1409d0837 (I will send PR soon)) it leaves behind:
```
admin       1344  0.4  1.2  21340 13544 ?        Ss   12:38   0:00 /usr/lib/systemd/systemd --user
admin       1387  0.0  0.0      0     0 ?        Zs   12:38   0:00 [dbus-broker-lau] <defunct>
admin       1393  0.0  0.0      0     0 ?        Z    12:38   0:00 [dbus-broker] <defunct>
```
If I wait for a bit, these processes are cleaned up and it all works fine. I wonder if it might help with https://github.com/cockpit-project/cockpit/issues/16970

Reply:
I like your new commit -- if there are leftover processes, then we at least know exactly when and how. Also, the running test looks very promising!